app/www/profile/index.php

```diff
@@ -18,14 +18,18 @@
 <div class="portal-body flexbox">
   <div class="profile-avatar">
     <img class="profile-avatar-img" src="/images/avatar/<?php echo $user['avatar'];?>.svg"/>
+  <a href="profile/customize/"><div class="customize">CUSTOMIZE ME!</div></a>
   </div>
   <div class="profile-details">
     <div>Hello,
       <span class="profile-detail"><?php echo $user['name'];?>!</span>
       <!-- <input type="text" name="firstname"><input type="text" name="lastname"> -->
     </div>
-    <div>Your birthday is on <span class="profile-detail"><?php echo $birthday_str;?></span></div> 
-    <div>You are in <span class="profile-detail"><?php echo $class_ord;?></span> class.</div>
+    <div>Your birthday is on: <span class="profile-detail"><?php echo $birthday_str;?></span></div>
+    <div>You are in <span class="profile-detail"><?php echo $class_ord;?></span> class.<p></p><p></p></div>
```

Comment:
Instead of using `<p></p>` to get spacing between text, it's a better idea to either use `<br/>`, or to apply a style and set the `margin` property to create separation.